<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogReg</class>
 <widget class="QWidget" name="LogReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>441</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>441</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HH聊天室</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/HHCharRoomIcon/chat.png</normaloff>:/HHCharRoomIcon/chat.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #a1c4fd, stop:1 #c2e9fb);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>14</number>
      </property>
      <item row="3" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" colspan="3">
       <layout class="QHBoxLayout" name="accounthorizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="accountLogLine">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="3">
       <layout class="QHBoxLayout" name="passwordhorizontalLayout">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLogline">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="LogText">
        <property name="styleSheet">
         <string notr="true"> QLabel {
        color: #333333; /* 文字颜色（深灰，不刺眼） */
        font-size: 13px;
        background: transparent; /* 透明背景，不遮挡窗口渐变 */
        padding: 2px 5px;
    }</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;HH Chat Room&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="regbtn">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Icon.qrc">
            <normaloff>:/HHCharRoomIcon/reg.png</normaloff>:/HHCharRoomIcon/reg.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Logbtn">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Icon.qrc">
            <normaloff>:/HHCharRoomIcon/log.png</normaloff>:/HHCharRoomIcon/log.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="Icon">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap resource="Icon.qrc">:/HHCharRoomIcon/cat.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutreg" rowminimumheight="3,0,0,0,0,0,0,0">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item row="6" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="repassLine">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
        </property>
        <property name="text">
         <string>密码确认</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="gologbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
        background-color: #ffffff;
        border: 2px solid #4285f4;
        border-radius: 12px; /* 圆角半径，可按需调整 */
        color: #4285f4;
        font-size: 14px;
        padding: 6px 20px; /* 上下内边距 6px，左右 20px，控制按钮大小 */
    }

    QPushButton:hover {
        background-color: #f0f7ff;
        border-color: #3367d6;
    }

    QPushButton:pressed {
        background-color: #e1ecff;
        border-color: #2854b4;
        /* 按下时轻微缩进效果，增强交互感 */
        padding: 7px 19px 5px 21px;
    }

    QPushButton:disabled {
        background-color: #f5f5f5;
        border-color: #ddd;
        color: #999;
    }</string>
          </property>
          <property name="text">
           <string>去登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="registrationbtn">
          <property name="styleSheet">
           <string notr="true">QPushButton {
        background-color: #ffffff;
        border: 2px solid #4285f4;
        border-radius: 12px; /* 圆角半径，可按需调整 */
        color: #4285f4;
        font-size: 14px;
        padding: 6px 20px; /* 上下内边距 6px，左右 20px，控制按钮大小 */
    }

    QPushButton:hover {
        background-color: #f0f7ff;
        border-color: #3367d6;
    }

    QPushButton:pressed {
        background-color: #e1ecff;
        border-color: #2854b4;
        /* 按下时轻微缩进效果，增强交互感 */
        padding: 7px 19px 5px 21px;
    }

    QPushButton:disabled {
        background-color: #f5f5f5;
        border-color: #ddd;
        color: #999;
    }</string>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="accountLine">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="passLine">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="MainTextreg">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> QLabel {
        color: #333333; /* 文字颜色（深灰，不刺眼） */
        font-size: 13px;
        background: transparent; /* 透明背景，不遮挡窗口渐变 */
        padding: 2px 5px;
    }</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="NicknameLine">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: rgba(255, 255, 255, 0.8);
    border: 1px solid #ddd;
    border-radius: 6px;
    padding: 5px;
}</string>
        </property>
        <property name="text">
         <string>昵称</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
